Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1194,6 +1194,16 @@
   categories:
     - UplinkMisc
 
+- type: listing
+  id: UplinkSuitAndTie
+  name: uplink-suitandtie-name
+  description: uplink-suitandtie-desc
+  productEntity: BoxSuitAndTie
+  cost:
+    Telecrystal: 1
```

Comment:
flash protection too cheap for 1 tc (mask is 2 tc iirc) remove the sunglasses or up the price

Reply:
SIR YES SIR OORAH